### ISSUE TITLE
Unable to install on CentOS 7

### ISSUE THREAD

**Issue by mikaelcom** [realm/realm-mobile-platform#59]
HI,
I can't install the realm object server due to nodejs version:
```
Resolving Dependencies
--> Running transaction check
---> Package realm-object-server-de.x86_64 0:1.0.0_BETA_4.2-336.el7 will be installed
Checking deps for realm-object-server-de.x86_64 0:1.0.0_BETA_4.2-336.el7 - u
looking for ('yaml-cpp', 'EQ', ('0', '0.5.1', None)) as a requirement of realm-object-server-de.x86_64 0:1.0.0_BETA_4.2-336.el7 - u
looking for ('nodejs', 'EQ', ('0', '4.5.0', None)) as a requirement of realm-object-server-de.x86_64 0:1.0.0_BETA_4.2-336.el7 - u
looking for ('openssl', None, (None, None, None)) as a requirement of realm-object-server-de.x86_64 0:1.0.0_BETA_4.2-336.el7 - u
looking for ('/bin/sh', None, (None, None, None)) as a requirement of realm-object-server-de.x86_64 0:1.0.0_BETA_4.2-336.el7 - u
realm-object-server-de-1.0.0_BETA_4.2-336.el7.x86_64 requires: nodejs = 4.5.0
--> Processing Dependency: nodejs = 4.5.0 for package: realm-object-server-de-1.0.0_BETA_4.2-336.el7.x86_64
Searching pkgSack for dep: nodejs
--> Finished Dependency Resolution
Dependency Process ending
Depsolve time: 0.443
Error: Package: realm-object-server-de-1.0.0_BETA_4.2-336.el7.x86_64 (realm_realm)
           Requires: nodejs = 4.5.0
           Installed: 1:nodejs-6.9.1-1.el7.x86_64 (@epel)
               nodejs = 1:6.9.1-1.el7
```

Isn't it possible to use a newer version? Or maybe pass the nodejs 4.5.0 executable bin so I don't have to downgrade the server. By the way I installed from the source nodejs 4.5.0 and it still fires this error.

Please get back to me,
regards,

**Comment by teotwaki** [comment]
Duplicate of #53 (even if it's not the same distribution).
